distributed/tests/test_spill.py

```diff
@@ -3,14 +3,15 @@
 import logging
 import os
 import uuid
+from tempfile import TemporaryDirectory
 
+import numpy as np
```

Comment:
Numpy is an optional dependency;  @jrbourbeau haas been working on a test bench without it for a while.
Please use pytest.importorskip instead the test instead.

Reply:
Thanks, I wasn't aware of this. Put some more thought into it and rewrote the relevant test to use the standard library's `array.array` instead.